.changeset/chilled-eyes-greet.md

```diff
@@ -0,0 +1,33 @@
+---
+'@commercetools-frontend/application-components': minor
+---
+
+We've updated the `title` property type of dialog components (`InfoDialog`, `FormDialog` and `ConfirmationDialog`) so, instead of only supporting `string` values, it will support `ReactNode` as well.
+This change is intended to allow for consumers to provide more complex content, such as custom components, to the `title` property of these components.
```

Comment:
```suggestion
We've updated the `title` property type of dialog components (`InfoDialog`, `FormDialog` and `ConfirmationDialog`) to be a `ReactNode` instead of `string` values.
This change is intended to enable consumers to define the title of the dialogs as a React component, thus allowing to support more use cases.
```